openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1250,7 +1250,7 @@ This specification also defines a new OAuth 2.0 Authorization Server metadata [@
 
 Credential Issuers often want to know what Wallet they are issuing Credentials to and how private keys are managed for the following reasons:
```

Comment:
Credential Issuers needs to know the Wallet that is receiving the Credentials and the security measures in place for managing private keys, for the following reasons: